README-en.md

```diff
@@ -4,29 +4,17 @@ Prevyo Semantic Services
 Tools for natural language processing.
 
 To use Prevyo Semantic Services: Web interface [https://pss.prevyo.com/](https://pss.prevyo.com/) and REST API are available. 
-To use the REST API you will need to create an Prevyo account on [https://pss.prevyo.com/](https://pss.prevyo.com/) then to contact [dev@emvista.com](mailto:dev@emvista.com) asking for an authentification token.
+To use the REST API you will need to create an Prevyo account on [https://pss.prevyo.com/](https://pss.prevyo.com/).
```

Comment:
you will need to create=> you must create
an Prevyo account => a Prevyo account